services/explorer-ui/src/hooks/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./useBlock";
```

Comment:
If you are going to have multiple hooks in this file I think it's better to rename the file to `services/explorer-ui/src/hooks/block.ts`